packages/core/src/functional/assertMcpController.ts

```diff
@@ -23,8 +22,9 @@ import type { IAgenticaController } from "../structures/IAgenticaController";
  */
 export async function assertMcpController<Model extends ILlmSchema.Model>(props: {
   name: string;
-  client: Client;
   model: Model;
+  // @ts-ignore
```

Comment:
please insert description about why to use @ts-ignore

I remember this is related to type script level dynamic type import.

This is because the compilation behavior changes when the library required for dynamic type import is installed and when it is not installed.

Reply:
yeah use `@ts-expect-error` instead @samchon

Reply:
Sadly, @ts-expect-error throws an error if no error is present.

In this case, since the implementation works both with and without a type error, @ts-ignore is appropriate.


https://github.com/wrtnlabs/agentica/pull/423